src/app/error/components/404.tsx

```diff
@@ -54,30 +24,63 @@ const Error404 = () => {
           textAlign: "center",
         }}
       >
-        <BoxWrapper>
-          <Typography variant="h1">404</Typography>
+        <Styled.BoxWrapper>
           <Typography
-            variant="h5"
-            sx={{ mb: 1, fontSize: "1.5rem !important" }}
+            variant="h1"
+            sx={{
+              mb: 2.5,
+              fontSize: { md: "2rem !important", xs: "1.5rem !important" },
+            }}
           >
-            Page Not Found ⚠️
+            Oops parece que te has perdido ⚠️
```

Comment:
I would remove the warning icon here to reduce the level of alarm of a user seeing this page.